docs/ui/components/Footer/PageVote.tsx

```diff
@@ -9,38 +9,40 @@ import { reportPageVote } from '~/providers/Analytics';
 export const PageVote = () => {
   const [userVoted, setUserVoted] = useState(false);
   return (
-    <div className="min-w-[200px]">
-      <CALLOUT theme="secondary" weight="medium">
-        Was this doc helpful?
-      </CALLOUT>
+    <div>
       {userVoted ? (
-        <CALLOUT theme="secondary" className="py-3">
+        <CALLOUT theme="secondary" className="py-1">
           Thank you for your vote! 💙
         </CALLOUT>
       ) : (
-        <div className="flex flex-row">
-          <Button
-            theme="secondary"
-            size="xs"
-            aria-label="Vote up"
-            className="mt-2.5 mx-1 min-w-[40px] text-center"
-            leftSlot={<ThumbsUpSolidIcon className="icon-sm" />}
-            onClick={() => {
-              reportPageVote({ status: true });
-              setUserVoted(true);
-            }}
-          />
-          <Button
-            theme="secondary"
-            size="xs"
-            aria-label="Vote down"
-            className="mt-2.5 mx-1 min-w-[40px] text-center"
-            leftSlot={<ThumbsDownSolidIcon className="icon-sm" />}
-            onClick={() => {
-              reportPageVote({ status: false });
-              setUserVoted(true);
-            }}
-          />
+        <div className="flex flex-row items-center gap-2 max-large:flex-col">
+          <CALLOUT theme="secondary" weight="medium" className="px-2">
+            Was this doc helpful?
+          </CALLOUT>
+          <div>
+            <Button
+              theme="secondary"
+              size="xs"
+              aria-label="Vote up"
+              className="mx-1 min-w-[40px] text-center"
+              leftSlot={<ThumbsUpSolidIcon className="icon-sm" />}
```

Comment:
Can we use non-solid icons, like `<ThumbsUpIcon />` here instead? Same for the thumbs down icon. (PS if this is out of scope we can ship it in another PR)